package.json

```diff
@@ -19,6 +19,7 @@
   "homepage": "https://github.com/ni-kismet/engineering-flot#readme",
   "devDependencies": {
     "concat": "^1.0.3",
+    "eslint": "^4.1.1",
```

Comment:
remove this. eslint is  requested and configured in webcharts-development-settings repo.